guide/english/html/attributes/required/index.md

```diff
@@ -4,7 +4,7 @@ title: Required
 ## Required
 
 
-The HTML `required` attribute is used in an input element to make the input field in a form required to submit the form. This means that the user must fill in the field. <br /> 
+The HTML `required` attribute specifies that an input field must be filled out before a form can be submitted. This means that the user must fill in the field. <br /> 
```

Comment:
```suggestion
The HTML `required` attribute specifies that an input field must be compulsorily filled out before a form can be submitted. <br /> 
```

Reply:
@thecodingaviator While this does shorten the text, `compulsorily` is not a word that everyone uses in their everyday life.  May be something like:
```suggestion
The HTML `required` attribute specifies that filling out an input field is mandatory before a form can be submitted. <br />
```

Reply:
@RandellDawson it definitely does. Compulsprily is not something many would be familiar with